### PR TITLE
Update family variant

### PR DESCRIPTION
**Pull request by Delphineray**

### CODE REVIEW

File: content/md/first-steps/manage-families.md [hunk 1]
```diff
@@ -183,12 +181,39 @@ Moreover, the attribute cannot be scopable, localizable and locale specific.
 The maximum number of variant axes for each variant level is less than **5**.
 :::
 
+Then, a screen to distribute the attributes between the different levels (`common attributes`, `variant attributes level 1` and `variant attributes level 2`) for the family variant opens
```
Comment (NolwennP) [inline content/md/first-steps/manage-families.md]: Missing dot at the end of the sentence.

File: content/md/first-steps/manage-families.md [hunk 2]
```diff
@@ -183,12 +181,39 @@ Moreover, the attribute cannot be scopable, localizable and locale specific.
...
+1.  You can remove an attribute from a variant level by clicking on the cross
+1.  A confirmation opens
+![Display a family variant](../img/Settings_Families_Variant_EditRemove.png)
+1.  Click on `OK` to confirm the remove
+1.  When you have made all your updates on the family variant attributes, click on `Save`
+![Display a family variant](../img/Settings_Families_Variant_Edit.png)
+
 ::: info
-This feature will be available soon. For the moment, you can only update the family variant by a dedicated import `Family variant import` available in CSV and XLSX.
+Move attributes from `common attributes` to the selected variant level or from the `variant attributes level 1` to the `variant attributes level 2` mean than these attributes will be now managed at a level under (not common but variant), the values for these attributes are kept and moved. For the `Clothing` example, if you move the attribute `Description` from the `common attributes` to the `variant attributes level 1`, it means than now the description can be managed by color.
```
Comment (NolwennP) [inline content/md/first-steps/manage-families.md]: **Moving** .... **means** **that** these attributes will now **be** managed
it means **that** now the description
(in bold, the words I changes or moved)

File: content/md/first-steps/manage-families.md [hunk 3]
```diff
@@ -183,12 +181,39 @@ Moreover, the attribute cannot be scopable, localizable and locale specific.
...
+
+You cannot move a variant axis or an attribute with the property `unique value` at `Yes`.
```
Comment (NolwennP) [inline content/md/first-steps/manage-families.md]: In a different warning block?

File: content/md/first-steps/manage-families.md [hunk 4]
```diff
@@ -183,12 +181,39 @@ Moreover, the attribute cannot be scopable, localizable and locale specific.
...
 :::
 
+::: warning
+Remove an attribute from a level means that this attribute will be managed as a `common attribute`, its values will be removed from the existing variant products with this family variant.
```
Comment (NolwennP) [inline content/md/first-steps/manage-families.md]: Removing

File: content/md/first-steps/manage-families.md (hunk 1 above)
Comment (Delphineray) [reply content/md/first-steps/manage-families.md]: yep

File: content/md/first-steps/manage-families.md (hunk 2 above)
Comment (Delphineray) [reply content/md/first-steps/manage-families.md]: ah bah oui that :)

File: content/md/first-steps/manage-families.md (hunk 3 above)
Comment (Delphineray) [reply content/md/first-steps/manage-families.md]: ok

File: content/md/first-steps/manage-families.md (hunk 4 above)
Comment (Delphineray) [reply content/md/first-steps/manage-families.md]: done

**Pull request merged**